components/base/select/Select.tsx

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import ReactSelect, { components } from "react-select";
-import arrowSelectIcon from "../../../public/images/arrow-select-icon.svg";
-import DoubleArrowSelectIcon from "../../../public/images/double-arrows-select-icon";
-import CheckMarkIcon from "../../../public/images/check-mark-icon";
+import dropdownArrowSingle from "../../../assets/images/component-icons/dropdown-arrow-single.svg";
+import { ReactComponent as DropdownArrowDouble } from "../../../assets/images/component-icons/dropdown-arrow-double.svg";
```

Comment:
This is a good example of the two ways to import SVGs now. The top is how you would import it if you wanted to use it as the src of an img tag, and the bottom is how you would import it if you wanted to use it as an inline SVG